src/libraries/Microsoft.Internal.Runtime.WindowsDesktop.Transport/src/Microsoft.Internal.Runtime.WindowsDesktop.Transport.proj

```diff
@@ -16,6 +16,16 @@
   <ItemGroup>
     <!-- Requires Private=true to calculate ReferenceCopyLocalPaths items.
          ReferringTargetFramework is set to $(NetCoreAppCurrent)-windows so that we pack the Windows specific implementation assemblies -->
-    <ProjectReference Include="@(WindowsDesktopCoreAppLibrary->'$(LibrariesProjectRoot)%(Identity)\src\%(Identity).csproj')" ReferringTargetFramework="$(NetCoreAppCurrent)-windows" PrivateAssets="all" Pack="true" Private="true" IncludeReferenceAssemblyInPackage="true" />
+    <ProjectReference Include="@(WindowsDesktopCoreAppLibrary->'$(LibrariesProjectRoot)%(Identity)\src\%(Identity).csproj');
```

Comment:
This feels like a lot of special casing here.  Is there any way to make it so that System.Diagnostics.EventLog.csproj can do the work to include the additional file via it's project reference?

Reply:
Can you please elaborate on why adding an additional ProjectReference is special? We do the same thing for analyzers in the aspnetcore transport package.

Message is intentionally marked as a private asset in EventLog and flowing that through as a transitive isn't possible. This is similar to analyzers which don't transitively flow either (because they aren't referenced).

Reply:
ping @ericstj

Reply:
Just that you're having to duplicate the notion that this internal file is part of the transport package.  It would be nice if we had a way for the library which needs this internal file to provide it so that we could keep this notion contained within the project that needs it (maybe a different target used when fetching package content, or as metadata on the existing item).  I can see how that's not that common though so take this as just a suggestion.

If you don't take that suggestion you could reduce the size of this change this by omitting this from WindowsDesktopCoreAppLibrary and just add the ProjectReference.  This file doesn't need to follow the rules that are defined for others in WindowsDesktopCoreAppLibrary (like exposing a reference and pinning version).

Reply:
If we don't need to pin the version of Messages then I'm fine with removing it from the list.

Reply:
If we don't need to pin the version of Messages then I'm fine with removing it from the list.